### ISSUE TITLE
Semantic Labeling Incorrect or Unassigned

### ISSUE THREAD

**Issue by ereio** [syphon-org/syphon#328]
- Send Button Labeling
- Back Button and Message Input Labeling
- Profile Screen Labeling Missing
